io/disk/softwareraid.py

```diff
@@ -45,69 +45,93 @@ def setUp(self):
         """
 
         smm = SoftwareManager()
-        if smm.check_installed("mdadm") is False:
-            print("Mdadm is not installed")
+        if not smm.check_installed("mdadm"):
+            self.skip("Mdadm must be installed before continuing the test")
```

Comment:
this code if SoftwareManager().install("mdadm") is False: never be going to execute as it check madm is installed if it not installed it skip the test